unique.c

```diff
@@ -0,0 +1,31 @@
+#include<stdio.h>
+#include<string.h>
+
+int main(){
+	printf("Print a string of characters.");
+	int x, z, y, b;
+	b=1;
+	x=0;
+	y=0;
+	char userinput[50];
+	fgets(userinput, 50, stdin);
+	z=strlen(userinput);
+	while(x<=z){
```

Comment:
This enters an infinite loop and doesn't print anything! your logic is off here - see if you can figure out where you went wrong. Also, for future reference, add comments, especially if your code isn't working so I can easily try and find where you went wrong.